test/oComments.test.js

```diff
@@ -30,29 +30,33 @@ describe("OComments", () => {
 		proclaim.equal(initSpy.called, false);
 	});
 
-	describe("new Comments(oCommentsEl, opts)", () => {
-		let boilerplate;
+	describe("when an authentication token is passed in", () => {
```

Comment:
@GlynnPhillips again, if you think this describe needs to be more clear, don't hesitate to change